<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="5" failures="0" name="SciPassTest.TestFunctionality-20150929153125" tests="5" time="0.006">
	<properties/>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
	<testcase classname="SciPassTest.TestFunctionality" name="test_bad_flow" time="0.001">
		<error message="'balancer'" type="KeyError">
<![CDATA[Traceback (most recent call last):
  File "/home/saikappa/workspace/SciPass/python/t/SciPassTest.py", line 194, in setUp
    config = str(os.getcwd()) + "/t/etc/SciPass.xml" )
  File "./SciPass.py", line 50, in __init__
    self._processConfig(self.configFile)
  File "./SciPass.py", line 514, in _processConfig
    config[dpid][name]['balancer'].addSensorGroup(group_info)
KeyError: 'balancer'
]]>		</error>
	</testcase>
	<testcase classname="SciPassTest.TestFunctionality" name="test_block_unknown_prefix" time="0.001">
		<error message="'balancer'" type="KeyError">
<![CDATA[Traceback (most recent call last):
  File "/home/saikappa/workspace/SciPass/python/t/SciPassTest.py", line 194, in setUp
    config = str(os.getcwd()) + "/t/etc/SciPass.xml" )
  File "./SciPass.py", line 50, in __init__
    self._processConfig(self.configFile)
  File "./SciPass.py", line 514, in _processConfig
    config[dpid][name]['balancer'].addSensorGroup(group_info)
KeyError: 'balancer'
]]>		</error>
	</testcase>
	<testcase classname="SciPassTest.TestFunctionality" name="test_bypass_unknown_prefix" time="0.001">
		<error message="'balancer'" type="KeyError">
<![CDATA[Traceback (most recent call last):
  File "/home/saikappa/workspace/SciPass/python/t/SciPassTest.py", line 194, in setUp
    config = str(os.getcwd()) + "/t/etc/SciPass.xml" )
  File "./SciPass.py", line 50, in __init__
    self._processConfig(self.configFile)
  File "./SciPass.py", line 514, in _processConfig
    config[dpid][name]['balancer'].addSensorGroup(group_info)
KeyError: 'balancer'
]]>		</error>
	</testcase>
	<testcase classname="SciPassTest.TestFunctionality" name="test_good_flow" time="0.001">
		<error message="'balancer'" type="KeyError">
<![CDATA[Traceback (most recent call last):
  File "/home/saikappa/workspace/SciPass/python/t/SciPassTest.py", line 194, in setUp
    config = str(os.getcwd()) + "/t/etc/SciPass.xml" )
  File "./SciPass.py", line 50, in __init__
    self._processConfig(self.configFile)
  File "./SciPass.py", line 514, in _processConfig
    config[dpid][name]['balancer'].addSensorGroup(group_info)
KeyError: 'balancer'
]]>		</error>
	</testcase>
	<testcase classname="SciPassTest.TestFunctionality" name="test_update_prefix_bw" time="0.001">
		<error message="'balancer'" type="KeyError">
<![CDATA[Traceback (most recent call last):
  File "/home/saikappa/workspace/SciPass/python/t/SciPassTest.py", line 194, in setUp
    config = str(os.getcwd()) + "/t/etc/SciPass.xml" )
  File "./SciPass.py", line 50, in __init__
    self._processConfig(self.configFile)
  File "./SciPass.py", line 514, in _processConfig
    config[dpid][name]['balancer'].addSensorGroup(group_info)
KeyError: 'balancer'
]]>		</error>
	</testcase>
</testsuite>
